draft-ietf-mls-protocol.md

```diff
@@ -2632,8 +2635,8 @@ A member of the group applies a Commit message by taking the following steps:
   any Update or Remove proposals, or if it's empty. Otherwise, the `path` value
   MAY be omitted.
 
-* If the `path` value is populated: Process the `path` value using the ratchet
-  tree the provisional GroupContext, to update the ratchet tree and generate the
+* If the `path` value is populated: Process the `path` value using the new ratchet
+  tree and the provisional GroupContext, to update the ratchet tree and generate the
```

Comment:
See comment from above. I think this part is indeed wrong and the broken syntax might be a hint that it's the result of a copy & pasta job gone wrong.

Reply:
This should be "provisional" in both cases -- the tree after applying proposals but before applying the UpdatePath.

Reply:
Uhm, yes.  Obviously it can't be the "new" ratchet tree, since this is the step that creates the "new" ratchet tree.  Not sure why I put "new" there.

Reply:
Oh, it's because we've only defined "old", "provisional", and "new" for GroupContext, and not for the ratchet tree.  I'll adjust the wording to fix that up.